test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -741,6 +741,15 @@ INI multiout_model(
   }
 );
 
+INI mol_attention(
+  "mol_attention",
+  {nn_base + "batch_size = 3",
+   sgd_base + "learning_rate = 1",
+   I("in0") + input_base + "input_shape = 1:1:256",
+   I("in1") + input_base + "input_shape = 1:84:256",
+   I("in2") + input_base + "input_shape = 1:1:5", // this shape should match mol_k
```

Comment:
If shape should match mol_k, can we use input_dim.width() to infer mol_k instead of setting it?

Reply:
It seems like mol_k should be able to interpret, but has been kept as a parameter as per the reference python implementation. If it is always fixed, will remove the property and interpret it from the input dimension later.

Reply:
I see, thanks for the explanation :)